.github/workflows/sync-redirects.yaml

```diff
@@ -13,20 +13,10 @@ on:
 
 jobs:
   sync:
-    # Prevent this job from running on forks.
-    if: github.repository_owner == 'nextstrain'
```

Comment:
You could preserve this `if` on the job if you wanted, even with the call of the centralized workflow.

Reply:
Added back in 96dde53db6385d74ee8d8b817d6a779049ce2142